recipe/meta.yaml

```diff
@@ -26,39 +26,18 @@ requirements:
     - wayland                            # [build_platform != target_platform]
     - libvulkan-headers                  # [build_platform != target_platform and not osx]
     - libvulkan-loader                   # [build_platform != target_platform and not osx]
-    - {{ cdt('mesa-libgl-devel') }}      # [linux]
-    - {{ cdt('mesa-libegl') }}           # [linux]
-    - {{ cdt('mesa-libegl-devel') }}     # [linux]
-    - {{ cdt('mesa-dri-drivers') }}      # [linux]
-    - {{ cdt('libdrm-devel') }}          # [linux]
-    - {{ cdt('libglvnd-glx') }}          # [linux]
-    - {{ cdt('libglvnd-egl') }}          # [linux]
-    - {{ cdt('libglvnd-devel') }}        # [linux]
   host:
     - qt6-main {{ version }}
     - wayland
-    - libvulkan-headers                  # [not osx]
-    - libvulkan-loader                   # [not osx]
+    - libgl-devel
+    - libegl-devel
+    - libopengl-devel
+    - libvulkan-headers
+    - libvulkan-loader
```

Comment:
i never got vulkan to work on OSX, did you?

Reply:
the recipe is linux-only, so this has no effect

Reply:
hmm smart....